diff -up shotwell-0.3.0/misc/shotwell-viewer.desktop.desktopfile shotwell-0.3.0/misc/shotwell-viewer.desktop
--- shotwell-0.3.0/misc/shotwell-viewer.desktop.desktopfile	2009-11-03 16:26:46.601060436 -0500
+++ shotwell-0.3.0/misc/shotwell-viewer.desktop	2009-11-03 16:27:11.099785337 -0500
@@ -9,6 +9,6 @@ Icon=shotwell
 Terminal=false
 NoDisplay=true
 Type=Application
-MimeType=image/jpeg
+MimeType=image/jpeg;
 Categories=Graphics;Viewer;Photography;GNOME;GTK;
 X-GIO-NoFuse=true
